build.assets/Dockerfile-centos7

```diff
@@ -1,17 +1,49 @@
-ARG RUST_VERSION
+## GIT2 ###################################################################
```

Comment:
I think `Dockerfile-centos7-fips` needs these changes as well

Reply:
Thanks. I forgot about it. I feel like our Dockerfiles multiplies every few months. I'll try to clean them up later, as I think we don't use most of them.